### PR TITLE
Make JUnit file name changeable

### PR DESCRIPTION
**Pull request by navartis**
Motivation - some services require a specific file name.

I thought it would be great if `xcbeautify` becomes more flexible.

Peace and goodness to all!

### CODE REVIEW

File: Sources/xcbeautify/Xcbeautify.swift [hunk 1]
```diff
@@ -29,6 +29,9 @@ struct Xcbeautify: ParsableCommand {
     @Option(help: "The path to use when generating reports")
     var reportPath = "build/reports"
 
+	@Option(help: "The the name of JUnit report file name")
+	var junitReportFileName = "junit.xml"
```
Comment (thii) [inline Sources/xcbeautify/Xcbeautify.swift]: Please fix indentation.

Comment (thii) [reply Sources/xcbeautify/Xcbeautify.swift]: Also, "filename" is a single word, so `junitReportFilename` is more correct than `junitReportFileName`.

Comment (navartis) [reply Sources/xcbeautify/Xcbeautify.swift]: @thii Fixed - you are welcome;)

File: README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Similar to `xcpretty`, but faster.
   bring anywhere. This also means less Ruby-dependant in your development
   environment and CI.
 
-**Note:** `xcbeautify` does not support generating JUnit or HTML test reports.
+**Note:** `xcbeautify` support generating JUnit reports.
```
Comment (danieleformichelli) [inline README.md:24]:
```suggestion
**Note:** `xcbeautify` supports generating JUnit reports. 
```

**Review by danieleformichelli — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (thii) [reply README.md:24]: This note feels unnatural now. I would suggest removing it completely.

**Pull request merged**